packages/kbn-cli-dev-mode/src/get_server_watch_paths.ts

```diff
@@ -28,6 +28,7 @@ export function getServerWatchPaths({ pluginPaths, pluginScanDirs }: Options) {
       (acc: string[], path) => [
         ...acc,
         Path.resolve(path, 'test/**'),
+        Path.resolve(path, 'integration_tests/**'),
```

Comment:
noticed that watcher restarts Kibana server when our integration tests write to a log file.